site/src/pages/notFound.js

```diff
@@ -0,0 +1,51 @@
+import { Panel } from "../components/styled/panel";
+import BreadCrumb from "../components/breadCrumb";
+import React from "react";
+import Layout from "../components/layout";
+import styled from "styled-components";
+import { Inter_14_500, Inter_18_700 } from "../styles/text";
+import { ReactComponent as Icon404 } from "../components/icons/404.svg";
+
+const StyledIcon404 = styled(Icon404)`
+  path {
+    fill: ${({ theme }) => theme.fontQuaternary};
+    fill-opacity: 1;
+  }
+`;
```

Comment:
recommended to create a icon js file like this, so we can use the icon directly without re-define the `StyledIcon404`

https://github.com/opensquare-network/statescan-v2/blob/main/site/src/components/icons/caretDownIcon.js

Reply:
This is fine, because we are not gonna to re-use Icon404 ever again somewhere else, this will be the only usage of it. That makes no point to extract it for re-usage.